packages/amplify-category-function/src/commands/function/remove.ts

```diff
@@ -21,6 +21,8 @@ module.exports = {
       resourceName = resourceToBeDeleted;
     }
 
+    resourceName = response.resourceName;
```

Comment:
Since we're assigning `resourceName` here, we can remove the previous assignment as it will always be overwritten

Reply:
sideeffect of late night fixes, I update it and put it to else branch, good catch!